src/renderer_vk.cpp

```diff
@@ -6020,26 +6020,26 @@ VK_DESTROY
 							streamMask >>= ntz;
 							idx         += ntz;
 
-							currentState.m_stream[idx].m_decl        = draw.m_stream[idx].m_decl;
-							currentState.m_stream[idx].m_handle      = draw.m_stream[idx].m_handle;
-							currentState.m_stream[idx].m_startVertex = draw.m_stream[idx].m_startVertex;
+							currentState.m_stream[idx].m_layoutHandle   = draw.m_stream[idx].m_layoutHandle;
```

Comment:
If it was `m_decl` it should be just `m_layout`

Reply:
Even if it is a VertexLayoutHandle?

Reply:
If you're fixing naming everywhere where `VertexLayoutHandle` is used then it's fine.

Reply:
yeah, every VertexLayoutHandle is handle or layoutHandle